bokehjs/src/coffee/models/glyphs/glyph.coffee

```diff
@@ -55,6 +55,18 @@ export class GlyphView extends BokehView
     bb = {minX: d.minX, minY: d.minY, maxX: d.maxX, maxY: d.maxY}
     return @_bounds(bb)
 
+  log_bounds: () ->
+    if not @index?
+      return bbox.empty()
+    log_x_index = rbush()
+    log_y_index = rbush()
+    positive_x_pts = @index.search(bbox.positive_x())
+    positive_y_pts = @index.search(bbox.positive_y())
+    log_x_index.load(positive_x_pts)
+    log_y_index.load(positive_y_pts)
+    bb = {minX: log_x_index.data.minX, minY: log_y_index.data.minY, maxX:log_x_index.data.maxX, maxY:log_y_index.data.maxY}
+    return bb
+
```

Comment:
@clairetang6 This is a clever use of rbush to find both the positive points in both half planes, then to find the max/min of each set. I definitely think the first use is great, because it uses an existing index. I do wonder about the use to find max/min, just because the overhead to both `load`  and then `search` the index might be more than doing a simple linear search of `positive_x_pts` and `positive_x_pts`. 

My only other comment is that I am not sure every glyph defines `@index` we should verify or catalog any that don't and add a warning or some kind of trivial if not-great fallback in those cases.